node-tests/blueprints/ember-bootstrap-test.js

```diff
@@ -182,8 +184,8 @@ describe('Acceptance: ember generate ember-bootstrap', function() {
 
   describe('install dependencies', function() {
 
-    const Blueprint = require('ember-cli/lib/models/blueprint');
-    const origTaskFor = Blueprint.prototype.taskFor;
+    let Blueprint = require('ember-cli/lib/models/blueprint');
```

Comment:
Just bikeshedding here, but I would prefer `const` when `require`ing. The second `let` makes sense though